src/course/syllabus/pre-apprenticeship-2/project.md

```diff
@@ -0,0 +1,5 @@
+Your project for the next two weeks is to build a _Projects Gallery_ using what you've learnt. The gallery should have an image of the project, a title and a short description.
+
+Add this to your learning site, either directly or via a link.
```

Comment:
I think we should explicitly tell them to build this as part of their existing GitHub learnings repo. Something like:

```suggestion
Create an `index.html` in your existing learnings repo on GitHub. You can create the gallery in here, with links to all the projects in the repo, and then deploy the repo to GitHub Pages.
```